sdk/servicebus/service-bus/samples/javascript/advanced/deferral.js

```diff
@@ -69,7 +70,7 @@ async function sendMessages() {
 async function receiveMessage() {
   const sbClient = new ServiceBusClient(connectionString);
 
-  // If receiving from a subscription you can use the createReceiver(topicName, subscriptionName) overload
+  // If receiving from a subscription, you can use the createReceiver(topic, subscription) overload
```

Comment:
It was intentional to use the `Name` suffix to the parameters here to make it clear to understand the function signature at a glance. Reason for removing it?

Reply:
Think of these as less "intentional changes" and more "we apparently only changed it in typescript but never synced the javascript".

I can change it so both have the JS comment. :)

Reply:
(it was in a few spots so those have changed as well).